client/state/selectors/is-eligible-for-signup-destination.js

```diff
@@ -18,14 +18,16 @@ import {
 	hasEcommercePlan,
 } from 'lib/cart-values/cart-items';
 import isEligibleForDotcomChecklist from './is-eligible-for-dotcom-checklist';
+import { retrieveSignupDestination } from 'signup/utils';
 
 /**
  * @param {Object} state Global state tree
  * @param {Number} siteId Site ID
  * @param {Object} cart object
+ * @param {String} destination The final destination URL after signup completes
```

Comment:
This `destination` param is no longer needed I think.

Reply:
Fixed in [0a1153f](https://github.com/Automattic/wp-calypso/pull/34156/commits/0a1153f8189218e80b4af70c2218422eda38f6f9).